src/org/openstreetmap/josm/gui/tagging/presets/TaggingPresetSearchPrimitiveDialog.java

```diff
@@ -67,7 +67,7 @@ public static synchronized TaggingPresetSearchPrimitiveDialog getInstance() {
         super(MainApplication.getMainFrame(), tr("Search for objects by preset"), tr("Search"), tr("Cancel"));
         setButtonIcons("dialogs/search", "cancel");
         configureContextsensitiveHelp("/Action/TaggingPresetSearchPrimitive", true /* show help button */);
-        selector = new TaggingPresetSelector(false, false, false);
+        selector = new TaggingPresetSelector(PresetSearchFilter.values());
```

Comment:
This would be the equivalent of new `TaggingPresetSelector(true, true, true)`, which I don't think you wanted. You probably wanted to just do new `TaggingPresetSelector()`.